sdk/core/azure-core/src/samples/java/com/azure/core/util/tracing/TracerJavaDocCodeSnippets.java

```diff
@@ -28,6 +28,14 @@ public void startTracingSpan() {
             updatedContext.getData(PARENT_SPAN_KEY).get());
         // END: com.azure.core.util.tracing.start#string-context
 
+        // BEGIN: com.azure.core.util.tracing.start#options-context
+        // start a new CLIENT tracing span with the given startt options and explicit parent span
+        StartSpanOptions options = new StartSpanOptions(StartSpanOptions.Kind.CLIENT);
```

Comment:
We should consider adding attributes here showcasing the full capabilities of options here.